contrib/nydusify/pkg/checker/rule/filesystem.go

```diff
@@ -301,11 +307,9 @@ func (rule *FilesystemRule) Validate() error {
 		return nil
 	}
 
-	image, err := rule.mountSourceImage()
-	if err != nil {
+	if err := rule.mountSourceImage(); err != nil {
```

Comment:
Where is the overlay mount? and how do we clean up these directories after checking?